recipes/google-differential-privacy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "differential-privacy" %}
+{% set version = "0.1" %}
+
+package:
+  name: "google-{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/master.tar.gz
+  sha256: ee3ee20fe61281e88337f2b1baee2fd322f7ae6aa28771972479ac93e57499dd
+
+build:
+  number: 0
+  script: bazel build differential_privacy/...
+
+requirements:
+  build:
+    - bazel
+    - {{ compiler('cxx') }}
+    - openjdk
+    - pip
+    - python
+    - postgresql
```

Comment:
Why is `postgresql` in `build`?

Reply:
just based off of the languages listed on the homepage, I was assuming they'd be needed